pkg/providers/vsphere/validator.go

```diff
@@ -179,51 +148,12 @@ func (v *Validator) ValidateClusterMachineConfigs(ctx context.Context, vsphereCl
 		}
 	}
 
-	if etcdMachineConfig != nil && etcdMachineConfig.Spec.OSFamily != anywherev1.Bottlerocket && etcdMachineConfig.Spec.OSFamily != anywherev1.Ubuntu {
-		return fmt.Errorf("etcd node osFamily: %s is not supported, please use one of the following: %s, %s", etcdMachineConfig.Spec.OSFamily, anywherev1.Bottlerocket, anywherev1.Ubuntu)
-	}
-
-	if etcdMachineConfig != nil && controlPlaneMachineConfig.Spec.OSFamily != etcdMachineConfig.Spec.OSFamily {
-		return errors.New("control plane and etcd machines must have the same osFamily specified")
-	}
-
-	if err := v.validateSSHUsername(controlPlaneMachineConfig); err == nil {
```

Comment:
Were these validations moved too in a different PR?

Reply:
No, I moved it [here](https://github.com/aws/eks-anywhere/blob/9192960b13e7a0fe95f40abfb2e43cac89e023be/pkg/api/v1alpha1/vspheremachineconfig.go#L137) in this same PR. I just moved it out of the method since it is just one validation that can stand alone like the others